SecuritySystemUWP/SecuritySystemUWP/AllJoynManager.cs

```diff
@@ -55,7 +55,8 @@ class AllJoynManager : ISecuritySystemService
             var images = await this.dropFolder.GetFilesAsync();
             var orderedImages = images.OrderByDescending(x => x.DateCreated);
             var file = orderedImages.FirstOrDefault();
-            if (null != file && file.Name != this.newestImage.Name)
+
+            if ((null != file) && (null == this.newestImage || file.Name != this.newestImage.Name))
```

Comment:
I don't think this does what you intend. Did you want to compare object identity or contents?